src/test/handler.test.ts

```diff
@@ -1,25 +1,32 @@
+import { createStubSourcegraphAPI } from '@sourcegraph/extension-api-stubs'
 import expect from 'expect'
 import mock from 'mock-require'
-import { createMockSourcegraphAPI, projects } from './stubs'
+import { projects } from './extension.test'
 
-const sourcegraph = createMockSourcegraphAPI()
-// For modules importing Range/Location/URI/etc
+export const sourcegraph = createStubSourcegraphAPI()
```

Comment:
Test files shouldn't export anything
```suggestion
const sourcegraph = createStubSourcegraphAPI()
```